File: despatch.Domain/Models/Default/Response/ContactDataPhoneNumberResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataPhoneNumberResponse : BaseEntityResponse
    {
        public string ContactDataId { get; set; }
        public string PhoneNumberId { get; set; }

        public virtual ContactDataResponse ContactData { get; set; }
        public virtual PhoneNumberResponse PhoneNumber { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/contact-data-phone-number-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataResponse } from './contact-data-response';
import { PhoneNumberResponse } from './phone-number-response';
export interface ContactDataPhoneNumberResponse extends BaseEntityResponse{
  contactData?: ContactDataResponse;
  contactDataId?: string;
  phoneNumber?: PhoneNumberResponse;
  phoneNumberId?: string;
}
